api/src/wfl/api/routes.clj

```diff
@@ -72,9 +72,9 @@
            :responses  {200 {:body ::spec/workflows}}
            :handler    handlers/get-workflows}}]
    ["/api/v1/workload/:uuid/retry"
-    {:post {:summary    "Resubmit workflows in workload by status."
-            :parameters {:path {:uuid   ::all/uuid}
-                         :body {:status ::all/status}}
+    {:post {:summary    "Resubmit workflows in workload by status and submission ID."
```

Comment:
TODO: Think this gets rendered in Swagger. Can I add more guidance here on how to use?  Maybe via `:description`?